thirdparty/README.md

```diff
@@ -473,6 +473,26 @@ Files extracted from the upstream source:
 - `LICENSE.txt` and `CHANGELOG.md`
 
 
+## OK Lab color space
+
+- Upstream: https://github.com/bottosson/bottosson.github.io/blob/master/misc/ok_color.h
+- Version: Master (d69831edb90ffdcd08b7e64da3c5405acd48ad2c, 2022)
+- License: MIT
+
+The license is merged with the header.
+
+Header includes are added.
```

Comment:
I don't see any added header includes actually. On the other hand I see you changed the `namespace` to `class`, is there a specific reason to make it a class?

Reply:
I don't remember. Can check.

Reply:
Changed to namespace.